infrastructure/modules/k3s/k3s.tf

```
@@ -1,16 +1,19 @@
 resource "ssh_resource" "install_primary_manager" {
-  host        = local.primary_manager.node.public_ip
-  user        = local.primary_manager.node.username
-  private_key = local.primary_manager.private_key
-  port        = 2244
+  host         = local.primary_manager.node.public_ip
+  user         = local.primary_manager.node.username
+  bastion_host = local.primary_manager.bastion_host
+  bastion_port = local.primary_manager.bastion_port
+  bastion_user = local.primary_manager.bastion_user
+  private_key  = local.primary_manager.private_key
+  port         = 2244
```

```suggestion
  host         = local.primary_manager.node.public_ip
  user         = local.primary_manager.node.username
  private_key  = local.primary_manager.private_key
  port         = 2244


  bastion_host = lookup(local.primary_manager, "bastion_host", null)
  bastion_port = lookup(local.primary_manager, "bastion_port", null)
  bastion_user = lookup(local.primary_manager, "bastion_user", null)
```

I think it'll make it a bit more readable to have these in a separate block. I've suggested a `lookup` so they can be optional, although I'm still working on how we achieve that in the module definition